tutorials/technology_demonstrations/arithmetic_expressions/arithmetic_expressions.ipynb

```diff
@@ -38,16 +38,16 @@
    },
```

Comment:
<p>Please remove the <code>|</code> in the <strong>text</strong> mathematical expression.</p><p> </p><p>"is solved with the synthesis engine....": please rephrase. What solves the mathematical expression is not the synthesis engine, but the execution part that executes the quantum program. The synthesis engine finds a quantum program the implements the mathematical expression. </p><p> </p><p>You can also mention that in both optimization scenarios the same result is obtained.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Classiq/classiq-library/pull/127/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>